content/docs/terraform-tools.mdx

```diff
@@ -0,0 +1,70 @@
+| [Sentinel](https://www.terraform.io/docs/cloud/sentinel/index.html) | HashiCorp |  Write and enforce fine-grained, logic-based policies for your infrastructure. Sentinel is an embedded policy-as-code framework integrated with the HashiCorp Enterprise products. | [Enforce Policy with Sentinel](https://learn.hashicorp.com/collections/terraform/policy) |
+
+
+## Develop Terraform Providers
+
+|Name | Maintainer |Description| Tutorial |
+|-----|------------|-----------|----------|
+| [Plugin SDK](/docs/extend/sdkv2-intro.html) | HashiCorp | Develop your own Terraform providers. | [Setup and Implement Read](https://learn.hashicorp.com/tutorials/terraform/provider-setup?in=terraform/providers) |
+|[Plugin Framework](/docs/plugin/framework/index.html) | HashiCorp | Develop your own Terraform providers. The framework provides improvements and additional features beyond what is available in the Plugin SDK. | [Implement Create and Read with the Terraform Plugin Framework](https://learn.hashicorp.com/tutorials/terraform/plugin-framework-create?in=terraform/providers) |
+|[`terraform-plugin-mux`](https://github.com/hashicorp/terraform-plugin-mux)| HashiCorp | Combine providers built using different SDKs and frameworks. This lets you upgrade between SDKs or frameworks on a resource-by-resource basis instead of all at once. | |
+| [`terraform-plugin-go`](https://github.com/hashicorp/terraform-plugin-go) | HashiCorp |  Use low-level Go bindings for the Terraform plugin protocol as a foundation for integrations. | |
+| [`terraform-plugin-go-contrib`](https://github.com/hashicorp/terraform-plugin-go-contrib) | HashiCorp |  Use this set of common code packages with `terraform-plugin-go`. |  |
+| [`terraform-plugin-log`](https://github.com/hashicorp/terraform-plugin-log) | HashiCorp | Attach context and information to logs and use multiple loggers to filter log output. | |
+
+
+## HashiCorp Products
+You can use HashiCorp products together to enable self-service automation across teams and organizations.
+
+|Name | Maintainer | Description| Tutorial |
+|-----|------------|-----------|----------|
+| [Vault provider](https://registry.terraform.io/providers/hashicorp/vault/latest/docs) | HashiCorp | Vault lets you secure, store, and tightly control access to tokens, passwords, certificates, encryption keys, and other sensitive data. Use this provider to read from, write to, and configure Vault.  | [Inject Secrets into Terraform Using the Vault Provider](https://learn.hashicorp.com/tutorials/cloud/terraform-hcp-provider-vault)|
+|[Consul Provider](https://registry.terraform.io/providers/hashicorp/consul/latest) | HashiCorp | [Consul](https://www.consul.io/) provides service-based networking for dynamic infrastructure. Use this provider to add external services to Consul, work with the key/value store, and more, using Terraform.  | [Use Terraform to Register External Services](https://learn.hashicorp.com/tutorials/consul/terraform-consul-provider)|
```

Comment:
Should we also plug the Boundary, HCP, HCS, and TFC/E providers?

https://registry.terraform.io/providers/hashicorp/boundary/latest
https://registry.terraform.io/providers/hashicorp/hcp/latest
https://registry.terraform.io/providers/hashicorp/hcs/latest
https://registry.terraform.io/providers/hashicorp/tfe/latest

Reply:
yes!! thank you so much - let me add these in :)

Reply:
Yes good call - I have added them!!